types/n3/n3-tests.ts

```diff
@@ -246,6 +262,11 @@ function test_doc_storing() {
     }
     console.log(mickey.subject, mickey.predicate, mickey.object, '.');
 
+    const quadGenerator: Generator<RDF.Quad> = store.readQuads(N3.DataFactory.namedNode('http://ex.org/Mickey'), null, null, null);
```

Comment:
```suggestion
    const quadGenerator: Generator<RDF.Quad> = store.match(N3.DataFactory.namedNode('http://ex.org/Mickey'), null, null, null);
```

Reply:
But isn't this update supposed to include the existence of `readQuads`? Hence the test should test that?

Reply:
See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65275#discussion_r1212515869